articles/azure-functions/functions-infrastructure-as-code.md

```diff
@@ -305,78 +350,172 @@ The Premium plan offers the same scaling as the Consumption plan but includes de
-                },
-                {
-                    "name": "FUNCTIONS_WORKER_RUNTIME",
-                    "value": "node"
-                },
-                {
-                    "name": "WEBSITE_NODE_DEFAULT_VERSION",
-                    "value": "~14"
-                },
-                {
-                    "name": "FUNCTIONS_EXTENSION_VERSION",
-                    "value": "~3"
-                }
-            ]
+  "type": "Microsoft.Web/sites",
+  "apiVersion": "2021-02-01",
+  "name": "[parameters('functionAppName')]",
+  "location": "[parameters('location')]",
+  "kind": "functionapp",
+  "dependsOn": [
+    "[resourceId('Microsoft.Web/serverfarms', variables('hostingPlanName'))]",
+    "[resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName'))]",
+    "[resourceId('Microsoft.Insights/components', variables('applicationInsightsName'))]"
+  ],
+  "properties": {
+    "serverFarmId": "[resourceId('Microsoft.Web/serverfarms', variables('hostingPlanName'))]",
+    "siteConfig": {
+      "appSettings": [
+        {
+          "name": "APPINSIGHTS_INSTRUMENTATIONKEY",
+          "value": "[reference(resourceId('microsoft.insights/components', variables('applicationInsightsName')), '2015-05-01').InstrumentationKey]"
+        },
+        {
+          "name": "AzureWebJobsStorage",
+          "value": "[concat('DefaultEndpointsProtocol=https;AccountName=', variables('storageAccountName'), ';EndpointSuffix=', environment().suffixes.storage, ';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName')), '2019-06-01').keys[0].value)]"
+        },
+        {
+          "name": "WEBSITE_CONTENTAZUREFILECONNECTIONSTRING",
+          "value": "[concat('DefaultEndpointsProtocol=https;AccountName=', variables('storageAccountName'), ';EndpointSuffix=', environment().suffixes.storage, ';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName')), '2019-06-01').keys[0].value)]"
+        },
+        {
+          "name": "WEBSITE_CONTENTSHARE",
+          "value": "[toLower(parameters('functionAppName'))]"
+        },
+        {
+          "name": "FUNCTIONS_EXTENSION_VERSION",
+          "value": "~4"
+        },
+        {
+          "name": "FUNCTIONS_WORKER_RUNTIME",
+          "value": "node"
+        },
+        {
+          "name": "WEBSITE_NODE_DEFAULT_VERSION",
+          "value": "~14"
         }
+      ]
     }
+  }
 }
 ```
 > [!IMPORTANT]
-> Don't set the [`WEBSITE_CONTENTSHARE`](functions-app-settings.md#website_contentshare) setting as it's generated for you when the site is first created.
+> Do not need to set the [`WEBSITE_CONTENTSHARE`](functions-app-settings.md#website_contentshare) setting as it's generated for you when the site is first created.
```

Comment:
Actually, contractions are recommended by our voice standards 😮  
```suggestion
> You don't need to set the [`WEBSITE_CONTENTSHARE`](functions-app-settings.md#website_contentshare) setting because it's generated for you when the site is first created.
```